data/brands/shop/furniture.json

```diff
@@ -1460,6 +1460,18 @@
         "shop": "furniture"
       }
     },
+    {
+      "displayName": "Ofiprix",
+      "id": "ofiprix-53177d",
+      "locationSet": {"include": ["es"]},
+      "matchTags": ["building/warehouse"],
```

Comment:
I've not seen this done for `building` before

Reply:
This changeset (https://www.openstreetmap.org/changeset/28786083) classified one of the stores as a "warehouse" so I thought that if someone is trying to classify it as such again, the brand should still come up and suggest a shop instead.

Note that usually Ofiprix stores keep large amounts of stock in display, so I guess the confusion comes from this.

In that specific case, in addition, the mapping is wrong as there's lots of other stores in the building, but this is another topic.

Reply:
```suggestion
```

Reply:
Fixed in 4973cc568f2cb6609630c5d9cfa88100dd9de75c